src/stories/Bubble.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Bubble from '@/domain/_common/components/Bubble';
+
+const meta = {
+  title: 'Example/Bubble',
+  component: Bubble,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+} satisfies Meta<typeof Bubble>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
```

Comment:
```suggestion
type Story = StoryObj<typeof Bubble>;
```
이 되어야할 것 같아요. 근데 meta로 해도 딱히 안돌아가진 않네요...🤔